app/views/posts/_expanded.html.erb

```diff
@@ -349,6 +349,7 @@
                     </div>
                     <div class="actions">
                       <%= f.submit 'Post', class: 'button is-outlined' %>
+                      <input type="button" name="button" value="Nope" class="button is-outlined" />
```

Comment:
To make this more consistent with elsewhere:

```html
<input type="button" name="cancel-comment" value="Cancel" class="button is-danger is-outlined" />
```

Reply:
Hmm.
* "Cancel" would be hard wired English in code. "Nope" is leet universal immune to l10n.
* Doesn't require `is-danger` because it is no danger.
* Agree on a different `name`.

Reply:
The whole of the application is hard-wired English - i18n is something we'll come to eventually, but not something we're worrying about right now. Whether it's "Cancel" or "Discard" or something, "Nope" doesn't make clear exactly what the button does, which is why I'm suggesting changing it.

Reply:
"Cancel" is no better than "Nope" besides being historic, maybe an historic error. "Discard" is very good, I will apply it.